inst/include/plant/interpolator.h

```diff
@@ -33,6 +33,7 @@ class Interpolator {
 
   // * R interface
   SEXP r_get_xy() const;
+	// change to const& vec?
```

Comment:
Decision on this?

Reply:
@devmitch

Reply:
It was in relation to the argument vector being passed in - I'm not exactly sure on how the translation from R to C++ is done so I'm not sure whether it would have any benefit, but I don't think it could hurt.